### PR TITLE
Template update for nf-core/tools v2.8

### PR DESCRIPTION
**Pull request by pinin4fjords**
Resolved conflicts for https://github.com/nf-core/differentialabundance/pull/121

Note: commented docker registry until the modules catch up in https://github.com/nf-core/modules/pull/3344.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -221,6 +246,10 @@ First, go to the [nf-core/differentialabundance releases page](https://github.co
 
 This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future. For example, at the bottom of the MultiQC reports.
 
+To further assist in reproducbility, you can use share and re-use [parameter files](#running-the-pipeline) to repeat pipeline runs with the same settings without having to write out a command with every single parameter.
```
Comment (jasmezz) [inline docs/usage.md:249]:
```suggestion
To further assist in reproducibility, you can use share and re-use [parameter files](#running-the-pipeline) to repeat pipeline runs with the same settings without having to write out a command with every single parameter.
```
Typo in template :scream: Can we change this sentence or will template tests complain?

File: nextflow.config [hunk 2]
```diff
@@ -229,46 +234,68 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     docker {
         docker.enabled         = true
+        //docker.registry        = 'quay.io'
```
Comment (jasmezz) [inline nextflow.config:241]: You deactivate it? Can you comment on why exactly?

**Review by jasmezz — APPROVED** [review]
LGTM!

File: nextflow.config (hunk 2 above)
Comment (jasmezz) [reply nextflow.config:241]: Ah probably because the modules without quay.io as default registry (like gunzip and untar) have not been updated yet...

Comment (pinin4fjords) [reply nextflow.config:241]: yep! Easy to reactivate in future

**Comment by pinin4fjords** [comment]
Thanks for the review @jasmezz

**Pull request merged**